### PR TITLE
Add Permission Role

### PR DESCRIPTION
**Pull request by sanglevinh**

### CODE REVIEW

File: frontend/src/pages/Account/index.tsx [hunk 1]
```diff
@@ -86,7 +87,10 @@ const Account = () => {
       </BoxFlex>
       <BoxFlex sx={{ justifyContent: 'space-between', mt: 10 }}>
         <ButtonSubmit onClick={onChangePwClick}>Change Password</ButtonSubmit>
-        <ButtonSubmit onClick={onDeleteAccountClick}>Delete Account</ButtonSubmit>
+        {
+          user && ROLE.ADMIN === user.role_id ?
```
Comment (ReiHashimoto) [inline frontend/src/pages/Account/index.tsx:91]: it's better to use selector(UserSelector.ts)

File: frontend/src/pages/Workspace/index.tsx [hunk 2]
```diff
@@ -103,15 +106,16 @@ const columns = (handleOpenPopupShare: () => void, handleOpenPopupDel: () => voi
           </ButtonCustom>
         ),
       },
+      user && [ROLE.ADMIN, ROLE.MANAGER].includes(user.role_id) &&
```
Comment (ReiHashimoto) [inline frontend/src/pages/Workspace/index.tsx:109]: define and use selector, too.

**Review by ReiHashimoto — CHANGES_REQUESTED** [review]

File: frontend/src/pages/Workspace/index.tsx [hunk 3]
```diff
@@ -103,15 +106,16 @@ const columns = (handleOpenPopupShare: () => void, handleOpenPopupDel: () => voi
...
       {
         field: 'share',
         headerName: '',
         minWidth: 90,
         renderCell: (params: GridRenderCellParams<string>) => (
-          params.row.owner !== "User 2" ?
-            <ButtonCustom onClick={handleOpenPopupShare}>
-              <SystemUpdateAltIcon sx={{transform: 'rotate(180deg)'}}/>
-            </ButtonCustom> : ""
+            params.row.owner !== "User 2" && user && ([ROLE.ADMIN, ROLE.MANAGER].includes(user.role_id)) ?
```
Comment (itutu-tienday) [inline frontend/src/pages/Workspace/index.tsx:115]: Workspace share is available to all authorized users.
*I am updating the following design document as there may have been some confusion in the explanation. Please check it.

- https://drive.google.com/drive/folders/1aiezc6vV16cxeLTFwKLaPdTrcYwyv3RQ
  - 権限構成設計_v0.5.xlsx

Comment (itutu-tienday) [inline frontend/src/pages/Workspace/index.tsx:115]: "User 2" is being compared and judged, but I think this is unnecessary processing, so please remove it.

**Review by itutu-tienday — CHANGES_REQUESTED** [review]
The remarks are noted. Please check.

File: frontend/src/pages/Workspace/index.tsx (hunk 2 above)
Comment (sanglevinh) [reply frontend/src/pages/Workspace/index.tsx]: @ReiHashimoto i fixed it

File: frontend/src/pages/Dashboard/index.tsx [hunk 4]
```diff
@@ -26,14 +30,17 @@ const Dashboard = () => {
...
           </LinkWrapper>
-          <LinkWrapper to="/console/account">
-            <BoxMenu>
-              <Box>
-                <AccountCircleIcon fontSize="large" />
-                <TitleMenu>Account</TitleMenu>
-              </Box>
-            </BoxMenu>
-          </LinkWrapper>
+          {
```
[the remark this answers is not in the document]
Comment (sanglevinh) [reply frontend/src/pages/Dashboard/index.tsx:31]: @itutu-tienday i fixed it

File: frontend/src/pages/Workspace/index.tsx (hunk 3 above)
Comment (sanglevinh) [reply frontend/src/pages/Workspace/index.tsx]: @itutu-tienday i fixed it

Comment (sanglevinh) [reply frontend/src/pages/Workspace/index.tsx]: @itutu-tienday i fixed it

File: frontend/src/pages/Account/index.tsx (hunk 1 above)
Comment (sanglevinh) [reply frontend/src/pages/Account/index.tsx]: @ReiHashimoto i fixed it

**Review by itutu-tienday — CHANGES_REQUESTED** [review]

File: frontend/src/pages/Dashboard/index.tsx [hunk 5]
```diff
@@ -31,8 +32,8 @@ const Dashboard = () => {
             </BoxMenu>
           </LinkWrapper>
           {
-            user && user.role_id === ROLE.ADMIN ?
-              <LinkWrapper to="/console/account">
+            isAdmin(user) ?
+              <LinkWrapper to="/console/account-manager">
```
[the remark this answers is not in the document]
Comment (sanglevinh) [reply frontend/src/pages/Dashboard/index.tsx:33]: @itutu-tienday 
I will fix it on branch 
https://github.com/arayabrain/optinist-for-server/tree/feature/account_manager

[the remark this answers is not in the document]
Comment (itutu-tienday) [reply frontend/src/pages/Dashboard/index.tsx:33]: @sanglevinh 
Thanks.
- If there is no problem, could you merge the current contents of #86 into #93?  
(I will reconfirm the action in #93)

**Comment by itutu-tienday** [comment]
@ReiHashimoto 
当PRへのレビュー指摘が対応されていますので、また確認をお願いします。
(Please check back as the review remarks to this PR have been addressed.)

File: frontend/src/pages/Dashboard/index.tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (sanglevinh) [reply frontend/src/pages/Dashboard/index.tsx:33]: @itutu-tienday I have merged it, please check it for me.
https://github.com/arayabrain/optinist-for-server/pull/93

**Review by ReiHashimoto — APPROVED** [review]
LGTM

**Review by itutu-tienday — APPROVED** [review]

**Pull request merged**